src/types/apl.ts

```diff
@@ -0,0 +1,11 @@
+export interface AuthData {
+  domain: string;
+  token: string;
+}
+
+export interface APL {
```

Comment:
This also I'd move to `src/APL/apl.ts` and near it, we can have `src/APL/local-env-apl.ts` and other implementations